tsconfig-backend.json

```diff
@@ -1,9 +1,8 @@
 {
-  "extends": "@tsconfig/node14/tsconfig.json",
+  "extends": "@tsconfig/node16/tsconfig.json",
   "include": ["api/**/*", "api-lib/**/*", "scripts/**/*"],
   "compilerOptions": {
-    "lib": ["es2021", "esnext"],
```

Comment:
esnext is tricky because it means different things depending on the Typescript version. So i figured avoiding its use would lead to more predictable behavior